internal/oci/runtime_oci.go

```diff
@@ -699,18 +699,30 @@ func (r *runtimeOCI) UpdateContainerStatus(c *Container) error {
 		return nil
 	}
 
-	cmd := exec.Command(r.path, rootFlag, r.root, "state", c.id) // nolint: gosec
-	if v, found := os.LookupEnv("XDG_RUNTIME_DIR"); found {
-		cmd.Env = append(cmd.Env, fmt.Sprintf("XDG_RUNTIME_DIR=%s", v))
+	var out []byte
+	var err error
+	// We have seen cases where this state call fails
+	// with little output, but the container is running and
+	// later calls succeed.
+	// There is little harm in trying again
+	for i := 0; i < 3; i++ {
+		out, err = r.attemptUpdateContainerStatus(c)
+		if err == nil {
+			break
+		}
+		logrus.Errorf("%v: attempt %d", err, i)
```

Comment:
could we add more context to this error?

Reply:
the context comes from the called function:
`failed to update container state for %s: %v", c.id, err`
I couldn't think of anything else to add out here that wouldn't be redundant. Is there anything you'd like to see added?